structure-tests.yaml

```diff
@@ -32,5 +32,5 @@ fileExistenceTests:
   shouldExist: true
 commandTests:
   - name: "application version"
-    command: "/usr/bin/identity-platform-admin-ui"
-    args: ["-version"]
+    command: "/usr/bin/identity-platform-admin-ui version"
+    # args: ["version"]
```

Comment:
I was expecting to use args....interesting it works without

Reply:
it was leftover from debugging, removed